readthedocs/builds/models.py

```diff
@@ -5,6 +5,8 @@
     absolute_import, division, print_function, unicode_literals)
 
 import logging
+
+from allauth.socialaccount.providers.github.views import GitHubOAuth2Adapter
```

Comment:
from should go after import, and third party import should go after stdlib import and before local imports